src/routes/productRouter.ts

```diff
@@ -108,5 +109,6 @@ router.get("/buyer-view-whishlist-product",userAuthorization(["buyer"]),isUserWi
 router.get("/buyer-view-whishlist-product/:id",userAuthorization(["buyer"]),isUserWishlistExistById,productController.buyerViewWishList)
 router.delete("/delete-whishlist-products",userAuthorization(["buyer"]),isUserWishlistExist,productController.buyerDeleteAllProductFromWishlist)
 router.delete("/delete-whishlist-product/:id",userAuthorization(["buyer"]),isUserWishlistExistById,productController.buyerDeleteProductFromWishList)
+router.post("/buyer-review-product/:id",userAuthorization(["buyer"]), isProductOrdered, productController.buyerReviewProduct )
```

Comment:
What is the use of this id parameter **`(:id)`**, I am not able to see where you have used it. And it was the one you were supposed to use.